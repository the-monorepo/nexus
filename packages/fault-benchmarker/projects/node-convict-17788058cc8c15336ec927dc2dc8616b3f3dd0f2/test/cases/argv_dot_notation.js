'use strict';

exports.conf = {
  server: {
    port: {
      default: 8080,
      format: 'port',
      arg: 'server.port'
    }
  },
  ui: {
    port: {
      default: 3000,
      format: 'port',
      arg: 'ui.port'
    }
  }
};

exports.argv = '--server.port 5000 --ui.port 4000';
