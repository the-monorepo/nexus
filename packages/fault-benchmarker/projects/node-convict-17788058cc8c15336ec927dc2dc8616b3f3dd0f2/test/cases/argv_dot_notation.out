{
  "server": {
    "port": 5000
  },
  "ui": {
    "port": 4000
  }
}